corehq/apps/settings/templates/settings/my_projects.html

```diff
@@ -7,23 +7,27 @@
   <div class="row-fluid">
     <table class="table table-striped table-bordered">
       <thead>
+      {% firstof show_session_timeout|yesno:"col-md-4,col-md-6" as col_class %}
```

Comment:
What's `firstof` doing here?  I wasn't familiar with the `yesno` filter, but it looks like that alone would translate `show_session_timeout` to the appropriate column class.

Reply:
> Or is that just a way of assigning it to a template var?

Exactly - it's a hack to assign a value to a variable. You can do it using the `with` tag but then it's only available inside the `with / endwith` block